### PR TITLE
[IC-364]  Add fn-elt variables

### PR DESCRIPTION
**Pull request by gquadrati**
### List of changes

 - Added 3 new Event Hub topic: `pdnd-io-cosmosdb-messages`, `pdnd-io-cosmosdb-message-status`, `pdnd-io-cosmosdb-message-status`
 - Added new tables for handling kafka failures: `fnelterrors_messages`, `fnelterrors_message_status`, `fnelterrors_notification_status`
 - Set new env variables for enabling ChangeFeed leases over `messages`, `message-status` and `notification-status` collections

### Motivation and context

Enable sending message data to PDND

### Type of changes

- [x] Add new resources
- [x] Update configuration to existing resources
- [ ] Remove existing resources

### Env to apply

- [ ] DEV
- [ ] UAT
- [x] PROD

### Does this introduce a change to production resources with possible user impact?

- [ ] Yes, users may be impacted applying this change
- [ ] No

### Does this introduce an unwanted change on infrastructure? Check terraform plan execution result

- [ ] Yes
- [ ] No

### Other information

---

### How to apply

After PR is approved
1. run deploy pipeline from Azure DevOps [io-infra.deploy](https://dev.azure.com/pagopaspa/io-iac-projects/_build?definitionId=218)
2. select PR branch
3. wait for approval

### CODE REVIEW

File: src/core/function_elt.tf [hunk 1]
```diff
@@ -74,13 +74,31 @@ module "function_elt" {
...
-    ERROR_STORAGE_ACCOUNT           = module.storage_account_elt.name
-    ERROR_STORAGE_KEY               = module.storage_account_elt.primary_access_key
-    ERROR_STORAGE_TABLE             = azurerm_storage_table.fnelterrors.name
-    COMMAND_STORAGE                 = module.storage_account_elt.primary_connection_string
-    COMMAND_STORAGE_TABLE           = azurerm_storage_table.fneltcommands.name
-    IMPORT_TOPIC_NAME               = "import-command"
-    IMPORT_TOPIC_CONNECTION_STRING  = module.event_hub.keys["import-command.io-fn-elt"].primary_connection_string
+
+    MESSAGES_TOPIC_NAME              = "pdnd-io-cosmosdb-messages"
+    MESSAGES_TOPIC_CONNECTION_STRING = module.event_hub.keys["io-cosmosdb-services.io-fn-elt"].primary_connection_string
```
Comment (fabriziopapi) [inline src/core/function_elt.tf:79]: Point to the wrong event hub topic. Should point to _pdnd-io-cosmosdb-messages.io-fn-elt_

File: src/core/function_elt.tf [hunk 2]
```diff
@@ -74,13 +74,31 @@ module "function_elt" {
...
+    MESSAGES_LEASES_PREFIX           = "messages-001"
+
+    MESSAGE_STATUS_TOPIC_NAME              = "pdnd-io-cosmosdb-message-status"
+    MESSAGE_STATUS_TOPIC_CONNECTION_STRING = module.event_hub.keys["io-cosmosdb-services.io-fn-elt"].primary_connection_string
```
Comment (fabriziopapi) [inline src/core/function_elt.tf:83]: ditto

File: src/core/function_elt.tf [hunk 3]
```diff
@@ -74,13 +74,31 @@ module "function_elt" {
...
+    MESSAGE_STATUS_LEASES_PREFIX           = "message-status-001"
+
+    NOTIFICATION_STATUS_TOPIC_NAME              = "pdnd-io-cosmosdb-message-status"
+    NOTIFICATION_STATUS_TOPIC_CONNECTION_STRING = module.event_hub.keys["io-cosmosdb-services.io-fn-elt"].primary_connection_string
```
Comment (fabriziopapi) [inline src/core/function_elt.tf:87]: ditto

**Review by fabriziopapi — CHANGES_REQUESTED** [review]

File: src/core/function_elt.tf (hunk 2 above)
Comment (gquadrati) [reply src/core/function_elt.tf]: Fixed in [d665f2f](https://github.com/pagopa/io-infra/pull/136/commits/d665f2f4fc9d9359a337a2fcea58e95b7675340d)

File: src/core/function_elt.tf (hunk 3 above)
Comment (gquadrati) [reply src/core/function_elt.tf]: Fixed in [d665f2f](https://github.com/pagopa/io-infra/pull/136/commits/d665f2f4fc9d9359a337a2fcea58e95b7675340d)

**Review by fabriziopapi — APPROVED** [review]

**Review by pasqualedevita — APPROVED** [review]

**Pull request merged**